maxprocs: No GOMAXPROCS change to reset (SIGSEGV: segmentation violation)

### What happened?

Latest docker image version `1.6-php8.4-alpine` produces a segmentation fault when just running as simple web-app. Building it with `1.5-php8.4-alpine` works.

Having the following extension active from my custom dockerimage.

```
install-php-extensions \
    amqp \
    curl \
    json \
    pdo \
    zip \
    pdo_mysql \
    redis \
    opcache \
    intl \
    gd \
    sockets \
    apcu \
    soap \
    exif
```

### Build Type

Docker (Alpine)

### Worker Mode

No

### Operating System

macOS

### CPU Architecture

Apple Silicon

### PHP configuration

```shell
default
```

### Relevant log output

```shell
{"level":"info","ts":1747646051.8465552,"msg":"maxprocs: No GOMAXPROCS change to reset"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x68 pc=0x10b5904]
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -19,12 +19,14 @@ RUN set -eux; \
 		/app/public \
 		/config/caddy \
 		/data/caddy \
+		/etc/caddy; \
 		/etc/frankenphp; \
 	sed -i 's/php/frankenphp run/g' /usr/local/bin/docker-php-entrypoint; \
 	echo '<?php phpinfo();' > /app/public/index.php
 
-COPY --link caddy/frankenphp/Caddyfile /etc/frankenphp/Caddyfile
-RUN curl -sSLf \
+COPY --link caddy/frankenphp/Caddyfile /etc/caddy/Caddyfile
+RUN ln /etc/caddy/Caddyfile /etc/frankenphp/Caddyfile && \
+	curl -sSLf \
 		-o /usr/local/bin/install-php-extensions \
 		https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions && \
 	chmod +x /usr/local/bin/install-php-extensions

--- a/alpine.Dockerfile
+++ b/alpine.Dockerfile
@@ -18,12 +18,15 @@ RUN set -eux; \
 		/app/public \
 		/config/caddy \
 		/data/caddy \
+		/etc/caddy \
 		/etc/frankenphp; \
 	sed -i 's/php/frankenphp run/g' /usr/local/bin/docker-php-entrypoint; \
 	echo '<?php phpinfo();' > /app/public/index.php
 
-COPY --link caddy/frankenphp/Caddyfile /etc/frankenphp/Caddyfile
-RUN curl -sSLf \
+COPY --link caddy/frankenphp/Caddyfile /etc/caddy/Caddyfile
+
+RUN ln /etc/caddy/Caddyfile /etc/frankenphp/Caddyfile && \
+	curl -sSLf \
 		-o /usr/local/bin/install-php-extensions \
 		https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions && \
 	chmod +x /usr/local/bin/install-php-extensions

